src/multirust-dist/build.rs

```diff
@@ -0,0 +1,22 @@
+use std::env;
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+fn main() {
+    let manifest_dir = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
+    let target = env::var("TARGET").unwrap();
+    let path = Path::new("src/TARGET");
```

Comment:
Oh build scripts should in theory never generate code directly into the source directory, you can throw it in `OUT_DIR` and then do something like:

```
let s = include_str!(concat!(env!("OUT_DIR"), "/target.txt")));
```